[2022-06-27 13:32:04,518] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-04-02T00:00:00+00:00 [queued]>
[2022-06-27 13:32:04,531] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-04-02T00:00:00+00:00 [queued]>
[2022-06-27 13:32:04,535] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:04,537] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:32:04,537] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:04,563] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_C> on 2022-04-02 00:00:00+00:00
[2022-06-27 13:32:04,570] {standard_task_runner.py:52} INFO - Started process 57623 to run task
[2022-06-27 13:32:04,588] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_C', 'scheduled__2022-04-02T00:00:00+00:00', '--job-id', '454', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpdnary8pe', '--error-file', '/tmp/tmppru3trsq']
[2022-06-27 13:32:04,594] {standard_task_runner.py:80} INFO - Job 454: Subtask model_C
[2022-06-27 13:32:04,706] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_C scheduled__2022-04-02T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:32:04,855] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-04-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-02T00:00:00+00:00
[2022-06-27 13:32:04,857] {python.py:173} INFO - Done. Returned value was: 4
[2022-06-27 13:32:04,895] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_C, execution_date=20220402T000000, start_date=20220627T133204, end_date=20220627T133204
[2022-06-27 13:32:04,961] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:32:05,015] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
